mmv1/products/dataplex/Datascan.yaml

```
@@ -336,7 +366,7 @@ properties:
               allow_empty_object: true
               send_empty_value: true
               description: |
-                ColumnAggregate rule which evaluates whether the column has duplicates.
+                Row-level rule which evaluates whether each column value is unique.
```

```suggestion
                Row-level rule which evaluates whether each row value is unique.
```

suggestion (?) unsure if read correctly. I could be wrong here